polkadot/runtime/common/src/slots/mod.rs

```diff
@@ -142,6 +158,11 @@ pub mod pallet {
 		ParaNotOnboarding,
 		/// There was an error with the lease.
 		LeaseError,
+		/// The signing account has no permission to do the operation.
+		NoPermission,
+		/// The signing account has privileges for this operation but the preconditions are not
+		/// met.
+		PreconditionNotMet,
```

Comment:
The context in which these errors arise is not meant to be called by signed accounts.
```suggestion
		/// The origin has no permission to do the operation.
		NoPermission,
		/// The origin has privileges for this operation but the preconditions are not met.
		PreconditionNotMet,
```